### DISCUSSION TITLE
I use Waitress to serve my Flask app on IIS, how can I use Flask-SocketIO too?

### DISCUSSION THREAD

**Discussion by geolectric** [miguelgrinberg/Flask-SocketIO#1587] [Q&A]
I use [Waitress](https://docs.pylonsproject.org/projects/waitress/en/stable/) to serve my Flask app on IIS.

> Waitress is meant to be a production-quality pure-Python WSGI server with very acceptable performance. It has no dependencies except ones which live in the Python standard library. 

I already start my app like this:
```
from waitress import serve
...
if __name__ == "__main__":
    serve(app, port=8443, url_scheme="https")
```

How would I use ```socketio.run(app)``` too? Do I just put it under serve()? Or will this not work?

**Comment by miguelgrinberg** [comment]
Does waitress support WebSocket? Without WebSocket there is really not a good match with Socket.IO.
